test/quantization/core/experimental/test_fake_quantize.py

```diff
@@ -51,7 +55,7 @@ def test_forward(self):
         X_reduced_precision_fp = apot_fake.forward(torch.clone(X), False)
 
         # get X_expected by converting fp -> apot -> fp to simulate quantize -> dequantize
-        X_to_apot = quantize_APoT(X, alpha, gamma, quantization_levels, level_indices)
+        X_to_apot, mask = quantize_APoT(X, alpha, gamma, quantization_levels, level_indices)
```

Comment:
I don't think we should be returning a mask

Reply:
@jerryzh168 do you think it makes more sense to just compute mask in the autograd forward method?